compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -435,7 +447,13 @@ impl<'a: 'ast, 'ast, 'tcx> LateResolutionVisitor<'a, '_, 'ast, 'tcx> {
             code,
         );
 
+        // FIXME: suggestions do not supported yet for delegation item
```

Comment:
Not sure why a special error is required here.
What happens if it is removed?

Reply:
Fully-qualified path will be suggested.  And this may be misleading, since we do not yet support type-relative paths.